sdk/identity/azure-identity/azure/identity/aio/_credentials/environment.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, **kwargs: "Any") -> None:
                 client_id=os.environ[EnvironmentVariables.AZURE_CLIENT_ID],
                 tenant_id=os.environ[EnvironmentVariables.AZURE_TENANT_ID],
                 certificate_path=os.environ[EnvironmentVariables.AZURE_CLIENT_CERTIFICATE_PATH],
+                password=os.environ.get(EnvironmentVariables.AZURE_CLIENT_CERTIFICATE_PASSWORD, None),
```

Comment:
`dict.get()` returns `None` by default
```suggestion
                password=os.environ.get(EnvironmentVariables.AZURE_CLIENT_CERTIFICATE_PASSWORD),
```